src/cluster/sharding.rst

```diff
@@ -296,5 +297,5 @@ any real gain. You might even get lower performance. As an example of this, we
 can take the author's (15 year) old lab server. It gets noticeably slower with
 more than one shard and high load, as the hard drive must seek more.
 
-How many shards you should have depends, as always, on your use case and your
+How many shards you should X depends, as always, on your use case and your
```

Comment:
Not sure what this change was meant to represent... "have" makes more sense to me?